AUV/auv_viewer_processing/auv_processing.py

```diff
@@ -0,0 +1,725 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*
+"""
+TODO
+-improve reporting function
+"""
+
+from datetime import datetime, timedelta
+from functools import partial
+from geopy.distance import vincenty
+from multiprocessing import Pool, cpu_count
+from netCDF4 import Dataset, num2date
+from osgeo import osr, gdal
+import StringIO
+import argparse
+import csv
+import operator
+import os
+import re
+import shutil
+import subprocess
+import sys
+import urllib2
+import uuid
+sys.path.insert(0, os.path.join(os.environ.get('DATA_SERVICES_DIR'), 'lib'))
+from python.imos_logging import IMOSLogging
```

Comment:
`lib/python` is already in the path, so you shouldn't need line 25, and can just say
```python
from imos_logging import IMOSLogging
```